Algorithms/Dynamic-Programming/LCS.cpp

```diff
@@ -3,112 +3,87 @@
 
 // Author: Pritam Negi.
 
-#include<bits/stdc++.h>
+#include <bits/stdc++.h>
```

Comment:
Please go through [guidelines](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards). I think use of `bits/stdc++.h` is not permitted. @pritamnegi

Reply:
Now take a look @monsij